### DISCUSSION TITLE
Can you please add some contact information for discussions on this project e.g. your email or some Chat forum.

### DISCUSSION THREAD

**Discussion by risaldar** [langroodi/Adaptive-AUTOSAR#6] [General]
Can you please add some contact information for discussions on this project e.g. your email or some Chat forum.
I need to select a baseline opensource adaptive AUTOSAR since I do not want to use a proprietary one for research purposes and your project came up as a candidate.

**Comment by langroodi** [comment]
I have already enabled the Discussion. You can ask your questions here.

**Reply by risaldar** [reply]
Thank you for enabling this feature. As I mentioned in the previous comment that I plan to have a master's thesis topic from the AUTOSAR domain. I think I am OK on the classical side but I want now to do the research on the Adaptive side.
It is unclear from ReadMe what was the background for this project and how do you plan to maintain it in the future, Is it your research-based project or something funded by some entity. Was there any specific use case in mind and so on?
do you think this port is stable enough for such purpose?

**Reply by langroodi** [reply]
The project is my own research (a personal one and not an academic one), and it is not sponsored by any party.

The scope of the project is not fully determined yet, but for instance the Update and Configuration Management (ara::ucm) is out of scope. Moreover, the ara::core will not be fully compatible with the standard because of the limitation of time.

It is a bit early to say, but when all the defined milestones are achieved, there won't be any further development/maintenance of the project. According to my timeline, it will be somewhere in ~18 months later.

As also indicated briefly in the ReadMe, the project will be an ARA simulator for x86-64 architectures and not a deploy-able program for ECUs. So, simulation-wise it should be stable. However, if you expect the project to be portable to an embedded system (based on e.g., RISC-V) with a certain extend of stability, I am afraid to say no.

**Reply by risaldar** [reply]
Thanks for sharing the information. Sad that you skipped UCM. I am on the concept team in AUTOSAR for Memory Stack Rework (aimed to cover FOTA and UCM) and It was one potential topic for thesis :(
Anyway, I will try to get things running in some shape and let you know when I get stuck.

**Comment by ajayjadhav900** [comment]
I have clone , build the entire repo and is built successfully.
./build/**adaptive_autosar** ./configuration/execution_manifest.arxml ./configuration/extended_vehicle_manifest.arxml ./configuration/diagnostic_manager_manifest.arxml ./configuration/health_monitoring_manifest.arxml

there is one typo error - adaptive_autosar exe is present in **build/bin** directory.

i have connected volvo page and configure the API and token 
still getting this error
 StateManagement Context ID:Lifetime;Context Description:Application lifetime logs;Log Level:Info;EM initial machine state transition is fetched successfully.
Mon May  6 11:47:59 2024
 StateManagement Context ID:Lifetime;Context Description:Application lifetime logs;Log Level:Info;EM is transited to the start-up state successfully.
Mon May  6 11:47:59 2024
 StateManagement Context ID:Lifetime;Context Description:Application lifetime logs;Log Level:Info;Execution state is reported successfully.
Mon May  6 11:48:01 2024
 ExtendedVehicle Context ID:Lifetime;Context Description:Application lifetime logs;Log Level:Error;**Setting the VIN failed due to unexpected RESTful response format.**

I have crossed checks ,API keys and token are valid.

Please assist on this.
